app/policies/predicates.rb

```diff
@@ -13,7 +13,12 @@ def responder_is_member_of_assigned_team?
   end
 
   def notify_responder_message_received
```

Comment:
This doesn't look like a predicate ... is this just a temporary home for it? Sounds like we need a module for state machine hooks.

Reply:
Agreed.  There are three types of hooks: 
 - predicates - which return true or false and are reference by if:
 - conditionals -which return a new state, or perhaps in the future a new workflow and are referenced by switch_workflow_using and transition_to_using
- hooks - after (or perhaps before) event procedures that need to be executed.

I think we need decent names and namespaces for them all.  Ideas?

Reply:
I think those names are good. I'd probably have thought that they could go into `ConfigurableStateMachine` but that's a bit of a mouthful. How about `Workflow`?

```
Workflow::Predicates
Workflow::Conditionals
Workflow::Hooks
```

Or, we might want:

```
Workflow::FOI::Standard::Predicates
Workflow::FOI::Standard::Conditionals
Workflow::FOI::Standard::Hooks
```